web/app/themes/gds/resources/blocks/tabs/tabs.blade.php

```diff
@@ -0,0 +1,24 @@
+@if( $innerBlocks ?? false )
+  <div {!! get_block_wrapper_attributes() !!}>
+    <div class="wp-block-buttons" role="tablist">
+      @foreach( $innerBlocks as $key => $tab )
+        <div class="wp-block-button" role="tab">
+          <button class="wp-block-button__link {{ $key === 0 ? 'active' : '' }}"
```

Comment:
Since we're managing `aria-selected` on this element I feel like we dont need the `active` class since we already have a selector we can use